PACKAGING.md

```diff
@@ -0,0 +1,39 @@
+# Shipping
+
+This readme file is for distribution packagers and provides some reasoning and hints about shipping the desktop client for a seamless experience on the Linux desktop.
+
+For that, ownCloud and the distributions need to collaborate.
+
+## Integration Problems
+
+The ownCloud desktop client gets shipped as AppImage because it is effortless to create, independent from the underlying distro and easy to install, as long as the distribution is not providing a package on their own which is frequently updated.
```

Comment:
```suggestion
The ownCloud Desktop App gets shipped as AppImage because it is effortless to create, independent from the underlying distro and easy to install, as long as the distribution is not providing a package on their own which is frequently updated.
```

Reply:
Where is this coming from? We ever since name it 'ownCloud Desktop Client' ... :confused:

Reply:
> Where is this coming from? We ever since name it 'ownCloud Desktop Client' ... 😕

This was decided about +2y ago by mstingl. Consequently I named this is all docs + versions. We now have this harmonized naming for the Desktop App, iOS App and Android App.

Reply:
Well but here we are in the client repo and talk about dev facing documentation.

Reply:
It's about consistency which is independent of where we are. just my 2c

Reply:
... but is anybody talking about apps in the context of desktop? :confused: